ui/src/components/debug_logger/mod.rs

```diff
@@ -34,49 +64,179 @@ pub fn DebugLogger(cx: Scope) -> Element {
+                                onpress: |_| {
+                                    filter_level.set(Level::Error);
+                                },
+                                tooltip: cx.render(rsx!(
+                                    Tooltip {
+                                        arrow_position: ArrowPosition::Top,
+                                        text: "Error".into()
+                                    }
+                                )),
+                            },
+                            Button {
+                                icon: Icon::Eye,
+                                appearance: Appearance::Secondary,
+                                onpress: |_| {
+                                    filter_level.set(Level::Trace);
+                                },
+                                tooltip: cx.render(rsx!(
+                                    Tooltip {
+                                        arrow_position: ArrowPosition::Top,
+                                        text: "Trace".into()
+                                    }
+                                )),
+                            },
+                        }
+                    })),
+                    Button {
+                        text: "State".into(),
+                        icon: Icon::Square3Stack3d,
+                        appearance: if *active_tab.get() == Tab::State { Appearance::Primary } else { Appearance::Secondary },
+                        onpress: |_| {
+                            active_tab.set(Tab::State);
+                        }
+                    },
+                    Button {
+                        text: "Web Inspector".into(),
+                        icon: Icon::ArrowTopRightOnSquare,
+                        appearance: Appearance::Secondary,
+                        onpress: |_| {
+                            window.webview.open_devtools();
+                        }
+                    },
+                },
+                div {
+                    class: "logger-nav-right",
+                    aria_label: "debug-logger-nav-right",
+                    IconElement {
+                        icon: if state.read().ui.theme.clone().unwrap_or_default().name == "Light" {
+                            Icon::Sun
+                        } else {
+                            Icon::Moon
+                        }
+                    },
+                    Switch {
+                        active: state.read().ui.theme.clone().unwrap_or_default().name == "Light",
+                        onflipped: move |_| {
+                            let current_theme = state.read().ui.theme.clone().unwrap_or_default();
+
+                            if current_theme.name != "Light" {
+                                let light_theme = get_available_themes().iter().find(|t| t.name == "Light").unwrap().clone();
```

Comment:
Is there a change of the themes not containing `Light`? If so, it might be better to avoid unwrapping to prevent panicking. Instead, we could probably either use `cloned()` on the `Option` it returns and supply it below or use a if-let condition. If it is going to be available, then it could be left as is but could do something like this 

```suggestion
                                let light_theme = get_available_themes().iter().find(|t| t.name == "Light").cloned().expect("theme is available");;
```